gui/src-tauri/src/main.rs

```diff
@@ -190,18 +194,19 @@ fn main() {
 		};
 
 		log::info!("Using Java binary: {:?}", java_bin);
-		let (recv, _child) = Command::new(java_bin.to_str().unwrap())
+		let (recv, child) = Command::new(java_bin.to_str().unwrap())
 			.current_dir(p)
 			.args(["-Xmx512M", "-jar", "slimevr.jar", "--no-gui"])
 			.spawn()
 			.expect("Unable to start the server jar");
+		_ = backend.insert(child);
```

Comment:
I didn't actually know this was possible, but I tried it in rust playground and it works.
@ImUrX do you have a style opinion on whether we should to it this way or instead use `let _ = `?

Reply:
huh, thats cool. Still I have some ideas on how to do this, I dont like the use of stdin for exiting but I understand it's mainly because of Windows limitations in signals :/